cloudgen_firewall/README.md

```diff
@@ -0,0 +1,81 @@
+# Agent Integration: cloudgen_firewall
+
+## Overview
+
+This integration monitors [cloudgen_firewall][1].
+
+## Setup
+
+### Installation
+
+The cloudgen_firewall check is included in the [Datadog Agent][2] package.
+Barracuda cloudGen Firewall installation is needed on your server.
+
+### Configuration
+
+1. Barracuda CloudGen Firewall with Administrative access.
+2. DataDog Agent installed and running(server or container that can receive syslog).
+3. Network Access between firewall and the dataDog agent (usually port 514 or custom).
+4. Syslog support enabled in Datadog Agent (tcp/udp listener configured).
+
+### Validation
+
+1. Confirm Datadog Agent is listening on the right port (eg., 514)
+    sudo netstat -tunlp | grep 514
+OR, for TCP/UDP listeners:
+    sudo lsof -i :514
+2. Confirm logs are reaching to the Agent and check for specific log source.
+    tail -f /var/log/datadog/syslog.log
+(If file doesn't exists, verify that syslog logs are being written by your config).
+3. Use tcpdump to confirm network traffic. On datadog Agent host:
+    sudo tcpdump -i any port 514
+You should see traffic from the cloudGen Firwewall IP address. if not, check firewall rules between cloudGen and Agent. Confirm the correct protocol (UDP/TCP) is being used on both sides.
+4. Check Live Tail in Datadog. Filter by source and service as defined in conf.yaml.
+5. Create a Test Log on the firewall by triggering an event.
+6. Check parsing and tagging in datadog.
```

Comment:
What should a user check for? Are there any [processors](https://docs.datadoghq.com/logs/log_configuration/processors/) that might be helpful for users here?